templates/web/components/app_nav_menu_items.html

```
@@ -9,7 +9,7 @@
 <ul class="menu">
   {% if request.team %}
   <li>
-    <a href="{% url 'ocs_notifications:notifications_home' request.team.slug %}" {% if active_tab == 'notifications' %}class="menu-active"{% endif %}>
+    <a href="{% url 'ocs_notifications:notifications_home' %}" {% if active_tab == 'notifications' %}class="menu-active"{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)